src/main/java/com/google/cloud/spanner/pgadapter/parsers/UuidParser.java

```diff
@@ -131,6 +135,15 @@ public static byte[] convertToPG(ResultSet resultSet, int position, DataFormat f
 
   @Override
   public void bind(ImmutableMap.Builder<String, Value> parametersBuilder, String name) {
-    parametersBuilder.put(name, Value.uuid(this.item));
+    // Send UUIDs to Spanner as untyped string values, so these can be used with both varchar and
```

Comment:
I think we should mention the reason as backward compatibility for this behaviour in the comment. 
It will be instantaneously clear to a new person.